backend/src/StdLibExecution/Libs/String.fs

```diff
@@ -993,6 +993,30 @@ let fns : List<BuiltInFn> =
         | _ -> incorrectArgs ())
       sqlSpec = NotYetImplemented
       previewable = Pure
+      deprecated = NotDeprecated }
+
+
+    { name = fn "ellipsis" 0
```

Comment:
I see this is also in a package. Can you remove it from Builtins?

Reply:
The package version treats EGCs as a single character rather than breaking them apart. Is that OK?